stable/nginx/nginx/templates/NetworkPolicy.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: nginx-{{ .Values.Instance }}-network-policy
```

```suggestion
  name: {{ template "nginx-fullname" . }}
```

Let's change this to conform to the object naming convention.